_posts/2017-09-05-life-at-codurance-robert-firek.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post 
+asset-type: post
+name: The Life of a Senior Software Craftsman, with Robert Firek  
+title: 'The Life of a Senior Software Craftsman, with Robert Firek'
+date: 2017-09-04 12:00:00 +00:00
+author: Robert Firek 
+image:
+  src: /assets/img/custom/blog/2017-09-01-do-we-have-a-good-safety-net-to-change-this-legacy-code.jpg
```

Comment:
This image might be changed. I don't know if we should have a specific image for these posts, or maybe any of these:
- https://pixabay.com/en/microphone-music-stage-event-298587/
- https://pixabay.com/en/office-meeting-business-partners-336368/
- https://pixabay.com/en/children-laugh-study-of-laptop-1822471/
- https://pixabay.com/en/children-win-success-video-game-593313/

Reply:
I like the microphone.  Thank you